Gruntfile.js

```diff
@@ -125,6 +125,10 @@ module.exports = function(grunt)
             'repository'
           ],
           overrides: {
+            main: ["js/kurento-utils.min.js"],
```

Comment:
npm mandates to not include minified files. bower allow to add them, but I think they require the main files to NOT be minified, since they could be minified by external tools, we add them as a convenience. Also, you have the file name hard-coded instead of using the ```pkg.name``` variable.